README.md

```diff
@@ -2,197 +2,230 @@
 
 [![Build Status](https://travis-ci.org/vtex/toolbelt.svg?branch=master)](https://travis-ci.org/vtex/toolbelt) [![npm version](https://img.shields.io/npm/v/vtex.svg?style=flat)](https://www.npmjs.com/package/vtex) [![Greenkeeper badge](https://badges.greenkeeper.io/vtex/toolbelt.svg)](https://greenkeeper.io/)
 
-All you need to start using the VTEX platform.
+The CLI that offers all you need to start using the VTEX platform efficiently.
+
+#### Summary
+- [How do we empower you?](#how-do-we-empower-you)
+- [First step: Installing the VTEX Toolbelt](#installing-the-toolbelt-with-yarn)
+- [Developing an app locally](#to-develop-locally-use-vtex-link)
+- [Customize your prompt](#wanna-know-something-cool-you-can-customize-your-prompt)
+- [FAQ](#frequently-asked-questions)
+- [Contributing](#contributing)
+- [Send us feedback](#tell-us-what-you-think-about-toolbelt)
+
+# How do we empower you? 
+
+VTEX Toolbelt is a command-line program that gives you access to all of 
+the features in the VTEX platform and increases your productivity, allowing you to:
+
+- **Manage your apps quickly:** List, install and uninstall available apps in your account;
+- **Work safely on your personal environments:** List, create and remove workspaces;
+- **Easily take changes to your final user:** Promote a workspace to `master`, making it public in your account URL;
```

Comment:
Maybe?
```suggestion
- **Easily ship changes to your final user:** Promote a workspace to `master`, making it public in your account URL;
```
Don't know which would be better though, I just think that `take changes to your final user` seems a little off